src/main/java/org/generationcp/middleware/api/germplasm/GermplasmAttributeServiceImpl.java

```diff
@@ -1,49 +1,56 @@
 package org.generationcp.middleware.api.germplasm;
 
-import org.generationcp.middleware.api.attribute.AttributeDTO;
 import org.generationcp.middleware.domain.germplasm.GermplasmAttributeDto;
 import org.generationcp.middleware.domain.germplasm.GermplasmAttributeRequestDto;
+import org.generationcp.middleware.domain.ontology.Variable;
 import org.generationcp.middleware.hibernate.HibernateSessionProvider;
 import org.generationcp.middleware.manager.DaoFactory;
+import org.generationcp.middleware.manager.ontology.api.OntologyVariableDataManager;
 import org.generationcp.middleware.pojos.Attribute;
-import org.generationcp.middleware.pojos.UDTableType;
+import org.generationcp.middleware.util.VariableValueUtil;
+import org.springframework.beans.factory.annotation.Autowired;
 
 import java.util.List;
-import java.util.Set;
-import java.util.stream.Collectors;
 
 public class GermplasmAttributeServiceImpl implements GermplasmAttributeService {
 
 	private final DaoFactory daoFactory;
 
+	@Autowired
+	private OntologyVariableDataManager ontologyVariableDataManager;
+
 	public GermplasmAttributeServiceImpl(final HibernateSessionProvider sessionProvider) {
 		this.daoFactory = new DaoFactory(sessionProvider);
 	}
 
 	@Override
-	public List<GermplasmAttributeDto> getGermplasmAttributeDtos(final Integer gid, final String attributeType) {
-		return this.daoFactory.getAttributeDAO().getGermplasmAttributeDtos(gid, attributeType);
+	//FIXME Should receive a filter object, to be addressed in IBP-4659
```

Comment:
IBP-4659 is marked resolved in JIRA, yet this FIXME remains. Is there another tech backlog ticket for this?